app/assets/config/manifest.js

```diff
@@ -2,3 +2,4 @@
 //= link_tree ../builds/images
 //= link_tree ../builds/
 //= link_tree ../images
+//= link_tree ../javascripts
```

Comment:
Shouldn't we be using `app/javascript` for consistency?

Reply:
yes